### PR TITLE
Support custom annotation and Stratum parameters. Specify fluentbit parser.

### PR DESCRIPTION
**Pull request by charlesmcchan**

### CODE REVIEW

File: stratum/values.yaml [hunk 1]
```diff
@@ -9,7 +9,14 @@ image:
...
   #   - aether-registry-credential
- 
+
+annotations:
+  fluentbit.io/parser: stratum
+
+extraParams:
+  - "-log_dir=/dev/null"
+  - "-v=0"
+  - "-stderrthreshold=0"
```
Comment (pudelkoM) [inline stratum/values.yaml:19]: Nitpick: for consistency I'd use either the longform `--flag` or the short form `-flag`. Both are supported, short is preferred.

Comment (charlesmcchan) [reply stratum/values.yaml:19]: Just to clarify, you want to use the short form for `-bf-sim`, `-chassis_config_file`, and all the `extraParams`?

Comment (pudelkoM) [reply stratum/values.yaml:19]: I'd prefer that, yes. Their all Stratum flags, but the different forms might lead somebody to think they are not.

**Comment by pudelkoM** [comment]
More flags from the discussion:

Disable P4 read/write request logging.
```
-write_req_log_file=""
-read_req_log_file="" (Stratum BFRT only at this time)
```

Limit the log file size to 1 MB.
```
-max_log_size=0
```

**Pull request merged**